devices/stm32mp153.yaml

```
@@ -22,6 +22,12 @@ TAMP:
   _strip:
     - "TAMP_"
 
+RCC:
+  RCC_BDCR:
```

It makes sense to strip all `RCC_` prefixes from register names.
```suggestion
RCC:
  _strip: RCC_
  BDCR:
```

---

I agree, but if possible, I'd like to make that change wholesale (for all registers and bitfields) in a future PR. I hesitate to make that change for this one register in this PR because that will leave us in an inconsistent state--currently, all registers for this platform are prefixed with the name of the peripheral. Interested to hear your thoughts!